src/lang/KnownTranslationKeys.php

```diff
@@ -128,6 +128,7 @@ final class KnownTranslationKeys{
 	public const COMMANDS_WHITELIST_REMOVE_SUCCESS = "commands.whitelist.remove.success";
 	public const COMMANDS_WHITELIST_REMOVE_USAGE = "commands.whitelist.remove.usage";
 	public const COMMANDS_WHITELIST_USAGE = "commands.whitelist.usage";
+	public const DEATH_ATTACK_ANVIL = "death.attack.anvil";
```

Comment:
This file is auto generated. Don't change it by hand.

Reply:
How would I generate another translation key then?

Reply:
https://github.com/pmmp/PocketMine-MP/blob/master/build/generate-known-translation-apis.php

Reply:
It has to be added to the English translation file first, and then the code generated with this script ^